### DISCUSSION TITLE
Get/Set object position

### DISCUSSION THREAD

**Discussion by matusko89** [robotology/community#18] [Q&A] [answered]
Hi all, I was wondering if it is possible to get and set the built-in blue sphere on the table. I am able to create a new object and move it around but it would be convenient if I could move also the blue built-in sphere. Is it possible, how? 

Thanks
Matus

**Comment by vtikha** [comment, marked as answer]
Hi @matusko89, yes, you can use the get/set functions on the built in blue sphere. Just use the following format (taken from the documentation). 

```
world get ball
world set ball 1.0 0.1 2.0
```

Best
Vadim
